spec/system/signed_in_user_creates_a_recipe_spec.rb

```diff
@@ -4,7 +4,7 @@
 require "support/features/clearance_helpers"
 
 RSpec.feature "A signed in user creates a recipe" do
-  scenario "with valid parameters" do
+  scenario "with valid parameters, but no picture" do
```

Comment:
I was reading these specs in the file, and I am not sure if I understand the success condition. Does this imply a fail case if there is no picture? How does the location of "/" imply fail or success? Thanks in advance for filling me in :)

I see that the spec wasnt created for this PR, so perhaps if you determine an adjustment for this, it can come in a follow-up PR.

I was comparing with the new spec below, which has expectations that make succss/fail more obvious to me.

Reply:
Thanks for pointing that out. I updated it to read more clearly now, so that the reader knows the picture is optional.